### ISSUE TITLE
DOnate pop-up does not dissapear after donation

### ISSUE THREAD

**Issue by avagp** [andrewbrg/codepad-chrome-app#48]
I don't know if this is a feature or not, but i have donated for Code Pad, and i still get the Donate Pop-up everytime i open a file.

There is a way to disable it?

**Comment by dickpountain** [comment]
Me too, just donated, expected pop-up to disappear. Is there a unlock code, registered version or something?

**Comment by avagp** [comment]
Let's hope the developer answers

Le sam. 29 août 2020 à 10:22, dickpountain <notifications@github.com> a
écrit :

**Comment by dickpountain** [comment]
Indeed, though I'm not holding my breath. The ChromeOS software world has a
lot to learn about support

Web: www.dickpountain.co.uk

**Comment by coder0107git** [comment]
The donate popup has some blue text that says "Please don't ask me again" and when it is clicked it disables the popup. 
![Screenshot 2022-08-22 102445](https://user-images.githubusercontent.com/76132013/185971141-15ef467e-6ad3-4b3c-ae91-d995a25b7d27.png)
